specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/examples/customApisGet.json

```diff
@@ -0,0 +1,34 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testResourceGroup",
+    "apiName": "testCustomApi",
+    "api-version": "2016-06-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "runtimeUrls": [
+            "https://tempuri.org",
+            "http://tempuri.org"
+          ],
+          "capabilities": [],
```

Comment:
The example should be in a way that capabilites is an array of strings. here it is ambiguos and not sure exactly what type of array it is.

Reply:
I don't know what is a legitimate capability, so for the time being I can put down "testCapability" to show off the type